src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Options/SingleValuedOption2`1.cs

```diff
@@ -8,23 +8,32 @@
 namespace Microsoft.CodeAnalysis.Options
 {
     /// <summary>
-    /// Marker interface for language specific options.
+    /// Marker interface for options that has the same value for all languages.
     /// </summary>
-    internal interface ILanguageSpecificOption : IOptionWithGroup
+    internal interface ISingleValuedOption : IOptionWithGroup
     {
+        /// <summary>
+        /// The language name that supports this option, or null if it's supported by multiple languages.
+        /// </summary>
+        /// <remarks>
+        /// This is an optional metadata used for:
```

Comment:
Also mention that the `LanguageName` is not used for computing the underlying option value. For example, if this property is null, so that multiple languages are supported by this option, still the underlying option will have a single value independent of the source language.